lib/plugins/orionPlugin.js

```diff
@@ -72,75 +68,6 @@ function translateAction(logger, originalAction) {
     return action;
 }
 
-/**
- * Extract the action from an XML body.
- *
- * @param {Object} logger        Instance of logops.Logger
- * @param {String} body          Raw string payload.
- */
-function inspectBodyXML(logger, body, callback) {
```

Comment:
XML in Orion was removed in version 1.0.0 (more than 8 years ago. It's a bit surprising this has survived so long in PEP code :)

NTC (informative)

Reply:
Keypass still uses XML, but not usually is exposed and then no needs a pep.

Reply:
You are right. I was referring to XML in the NGSI API. The rests of XML support in PEP (Keypass, etc.) has been preserved.